static/js/subs.js

```diff
@@ -868,10 +868,11 @@ exports.open_create_stream = function () {
 
 
 exports.sub_or_unsub = function (sub, row_object) {
```

Comment:
Now, that we are calculating the stream_row from the sub object, I think the row_object is not required and this parameter can be removed. Take a look at stream_edit.js https://github.com/zulip/zulip/blob/2825e6ad480dc2f879b99c12c3339ca2c31bc1a1/static/js/stream_edit.js#L640
I think we can remove the e.currentTarget passed to the sub_unsub function too.

Reply:
All these changes were made by me, while adding the spinner and I think can be removed safely( But it would probably be better to check once)